servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/QuarkusNessieAuthenticationConfig.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.projectnessie.server.config;
+package org.projectnessie.quarkus.config;
```

Comment:
Shouldn't this stay in `quarkus-server` because it is only used by the quarkus http authenticator?

Reply:
Yea - doesn't make sense. Moved back.